src/Microsoft.DncEng.SecretManager/SecretPropertiesWrapper.cs

```diff
@@ -0,0 +1,13 @@
+namespace Microsoft.DncEng.SecretManager;
+
+public class SecretPropertiesWrapper
+{
+    public SecretProperties Properties { get; }
+    public bool NextRotationFound { get; }
+
+    public SecretPropertiesWrapper(SecretProperties properties, bool nextRotationFound) 
```

Comment:
Why do we define a wrapper instead of including the information about the next rotation directly in the secret properties? This looks like unnecessary layer, also the naming "*Wrapper" is pretty generic.